### PR TITLE
[PERFSCALE-2539] Renames the jobs in OVN-IC

### PR DESCRIPTION
**Pull request by krishvoor**
This PR renames the jobs and adds control-plane testing in OVN-IC
for better coverage/testing.

### CODE REVIEW

**Review by venkataanil — APPROVED** [review]
LGTM
Thanks @krishvoor for adding node-density along with cluster-density (i.e replacing openshift-qe-cluster-density-v2 with openshift-qe-control-plane). Can you please add here the jira ticket which I reported for this requirement?
@jtaleric @svetsa-rh @paigerube14 Can we merge this PR so that node-density tests will also be run along with cluster-density in next periodic run?

**Comment by krishvoor** [comment]
> LGTM Thanks @krishvoor for adding node-density along with cluster-density (i.e replacing openshift-qe-cluster-density-v2 with openshift-qe-control-plane). Can you please add here the jira ticket which I reported for this requirement? @jtaleric @svetsa-rh @paigerube14 Can we merge this PR so that node-density tests will also be run along with cluster-density in next periodic run?

Hi @venkataanil I have added the JIRA details as part of the PR Title and this will reflect in the JIRA as well.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-ovnic.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-aws-ci-cdv2-test-252nodes-scale-ovnic
+- as: perfscale-aws-ci-control-plane-test-252nodes-scale-ovnic
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-ovnic.yaml:24]: We lose history when the tests are renamed. For the sake of traceability and with these ovnic tests being temporary, I request that we don't change any of the job names.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-ovnic.yaml [hunk 2]
```diff
@@ -37,9 +37,9 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    workflow: openshift-qe-cluster-density-v2
+    workflow: openshift-qe-control-plane
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-ovnic.yaml:40]: cdv2 alone took 1h45m recently. A rehearse would tell us, but we also likely need to set higher timeouts in the refs to make sure we don't timeout.

**Comment by svetsa-rh** [comment]
@krishvoor any reason for renaming ovnic jobs? Last time we spoke about the naming inconsistency, Joe specifically said can we leave these as exceptions since they are one time tests.

**Comment by jtaleric** [comment]
> LGTM Thanks @krishvoor for adding node-density along with cluster-density (i.e replacing openshift-qe-cluster-density-v2 with openshift-qe-control-plane). Can you please add here the jira ticket which I reported for this requirement? @jtaleric @svetsa-rh @paigerube14 Can we merge this PR so that node-density tests will also be run along with cluster-density in next periodic run?

Why do we need node-density and node-density-cni @venkataanil ?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-ovnic.yaml (hunk 1 above)
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-ovnic.yaml:24]: @svetsa-rh @afcollins Prow jobs can only be 61 characters long and `ocp-qe-perfscale-aws-ci-control-plane-test-252nodes-scale-ovnic` does not make the cut,

> with these ovnic tests being temporary, I request that we don't change any of the job names.
Ack, will keep the job name relevant to the workload we are running, and will find a way to keep it <=61 characters long.

**Comment by venkataanil** [comment]
node-density is for latency calculation. I am getting queries about what is the podready latency on 252 nodes. This test can help to answer those queries. It also helps in identifying regressions  in latencies because of nightlies  at higher node count.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-ovnic.yaml (hunk 1 above)
Comment (jtaleric) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-ovnic.yaml:24]: I want to push back on running node-density and CNI.

**Comment by jtaleric** [comment]
> node-density is for latency calculation. I am getting queries about what is the podready latency on 252 nodes. This test can help to answer those queries. It also helps in identifying regressions  in latencies because of nightlies  at higher node count.  

can we report node-density-cni as it is more relevant to network tests?

**Comment by venkataanil** [comment]
For a long time I am reporting latency from both node-density and node-density-cni. Lets continue reporting both as latencies with node-density is also very important. These jobs are for a short term till OVN-IC becomes mature. Earlier I used to run regularly in airflow and now I am missing the data as it is taking time for these specific OVN-IC jobs (which are temporary) to get merged in prow. Kindly merge these jobs which helps in detecting regressions in OVN-IC in early stages.

**Comment by krishvoor** [comment]
@venkataanil @jtaleric closing this PR, as we have migrated towards 4.15 based OVNIC

**Pull request closed without merging**